src/main/scala/cromwell/engine/package.scala

```diff
@@ -43,6 +43,8 @@ package object engine {
   type CallOutputs = Map[LocallyQualifiedName, CallOutput]
   type HostInputs = Map[String, WdlValue]
 
+  class CromwellFatalException(exception: Throwable) extends Throwable(exception)
```

Comment:
This should extend `Exception`, or even `RuntimeException`, but not `Throwable`.